easybuild/easyconfigs/m/MONAI/MONAI-1.3.0-foss-2023a.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'MONAI'
+version = '1.3.0'
+
+homepage = 'https://monai.io/'
+description = """
+MONAI is a PyTorch-based, open-source framework for deep learning in healthcare
+imaging, part of PyTorch Ecosystem.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+github_account = 'Project-MONAI'
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('PyTorch', '2.1.2'),
+    ('PyTorch-Ignite', '0.4.11'),  # Monai 1.3.0 requires PyTorch-Ignite==0.4.11
```

Comment:
Since the installation of `ignite` is rather simple, we can remove this dependency and add this specific version as an extension to this easyconfig.